cmd/vulcan-trivy/vulcan-trivy.go

```diff
@@ -204,6 +209,15 @@ func run(ctx context.Context, target, assetType, optJSON string, state checkstat
 		}
 		trivyArgs = append(trivyArgs, []string{"--scanners", sc}...)
 
+		if opt.ImageChecks.Secret {
+			if !opt.DisableCustomSecretConfig {
+				trivyArgs = append(trivyArgs, []string{"--secret-config", "secret.yaml"}...)
```

Comment:
```suggestion
				trivyArgs = append(trivyArgs, "--secret-config", "secret.yaml")
```

Reply:
I agree with the suggestion, but I was just trying to follow the same constructions that were already made in the check.